autopush_rs/src/client.rs

```diff
@@ -271,7 +276,13 @@ where
                         uaid,
                         use_webpush: Some(true),
                         ..
-                    } => uaid,
+                    } => {
+                        if let Some(uaid) = uaid {
```

Comment:
this could be `uaid.and_then(|uaid| Uuid::parse_str(uaid.as_str()).ok())`

use_webpush: Some(true) isn't necessary in the destructuring above either is it

Reply:
Thanks! I knew there was something shorter I was missing.

Reply:
I think the Some(true) is needed to enforce that it was set to True?

Reply:
I guess but it's never false anyway